### ISSUE TITLE
Enable dataset peer-review

### ISSUE THREAD

**Issue by kbraak** [gbif/ipt#1318]
Dataset peer-review is an essential step in the data publishing workflow that can enhance its quality and its fitness for use.

It would be helpful, if the IPT could facilitate the peer-review process. 

The **publisher** would be able to request a peer-review of their dataset sending an invitation to one or more individuals. 

A simple checklist could guide the **reviewer** through the review process, for starters to help ensure the data is complete, meaning it contains valid answers to the [five Ws](https://github.com/gbif/ipt/wiki/dataQualityChecklist#introduction).

Following successful review, the dataset could receive a 'reviewed badge'. It would be possible to clearly differentiate peer-reviewed datasets from non-peer-reviewed datasets when browsing the IPT's list of datasets. 

In exchange for their hard work, the **reviewer(s)** would be credited as a 'reviewer' in the dataset's list of associated parties. In some cases, the **reviewer(s)** could even receive financial compensation for their efforts, because a thorough review requires significant time and expertise. 

---
Note: the root of this idea was an outcome from a recent NSG breakout group meeting with @andrejjh, @siro1, @ahahn-gbif, etc

**Comment by peterdesmet** [comment]
Yes please!

**Comment by kbraak** [comment]
Related to this issue, note that the new roles 'reviewer' and 'mentor' are soon going to added to the [EML Agent Role Vocabulary](http://rs.gbif.org/vocabulary/gbif/agent_role.xml). For more information [see this pull request in rs.gbif.org](https://github.com/gbif/rs.gbif.org/pull/6#issuecomment-249145120).

**Comment by kbraak** [comment]
Some helpful related resources offering guidance on how to implement data peer-review:

* [Lawrence et al., “Citation and Peer Review of Data,” 13–15.](http://www.ijdc.net/index.php/ijdc/article/viewFile/181/265) has a data review checklist that could be adapted for use by the IPT.
* [Curating Research Data, Volume Two: A Handbook of Current Practice](https://t.co/3xG4wQ0IOn) has a section on peer review of data entitled "8.2 Facilitate the Peer Review of Data"

**Comment by dschigel** [comment]
To clarify, seems like we are talking about optional post-publication peer review system, and some inspiration could be taken from http://riojournal.com/about#HowItWorks. Alternatively, same pages describe pre-submission option for the peer review, which some see as potential publication blocker, others as data quality driver. See also https://www.peerageofscience.org/how-it-works/process-flow/. What if there would be a hub for individuals to pre-submit, to peer-review and to tag datasets, so then publishers would pick the ready datasets and publish?

**Comment by kbraak** [comment]
Collecting more ideas.. 

At [RDA9](https://www.rd-alliance.org/multimedia-publishing-and-data-visualization-and-scientific-data-repositories), I saw a presentation about the [RADAR data repository](https://www.radar-projekt.org/display/RD/Home), which has an optional pre-publication peer review feature. They designed the peer review to be a simple as possible. The way it works, is the dataset is frozen for the duration of the review and a secure 'review URL' is generated that allows reviewers to look at the unpublished dataset. It is then up to the publisher to share the URL with peer reviewers of their choice.

**Comment by dschigel** [comment]
@kbraak I like the idea to have it, but keeping it simple. Perhaps, should not be preventing to sharing data, but option to have a data quality star as peer-reviewed.

**Comment by kbraak** [comment]
Thanks @dschigel. 

A [recent survey by Todd Carpenter](https://scholarlykitchen.sspnet.org/2017/04/11/what-constitutes-peer-review-research-data/) looked at how the data peer review recommendations from [Lawrence et al.](http://www.ijdc.net/index.php/ijdc/article/viewFile/181/265) were being adopted by of 39 journals that publish data papers, by examining their data peer review instructions. Carpenter found that peer review is more focused on the overall quality of the metadata, perhaps because it is the easiest to review objectively. He points out that focusing on the quality of the data itself is particularly challenging to undertake in practice at scale, because few reviewers have the time or expertise to perform it fully. Thanks @kcopas for bringing this survey to my attention.

**Comment by siro1** [comment]
I also found Todd Carpenter's review interesting. 
Looking at data paper review in two parts, namely metadata review and review of the data itself might be the way to go. It may be difficult to find people that can review both the metadata and the data itself at the same time.
-Metadata review can be done by a subject expert that may not necessarily be a data scientist. This would be a one time process.
-Data quality review can be done by a data scientist or technician that may not necessarily be a subject expert. This can be repeated over time since datasets may not be static. The review of the data itself could also be an automated process since it's not practical for a human to scrutinize every record in a very large or complex dataset. The automated process could generate reports giving quality scores. I can imagine that a dataset might be published with a low score at first but the quality score could improve over time if the data owners and other interested parties work on it to correct errors and add missing information. 
Todd Carpenter did not mention this reference http://www.libellarium.org/index.php/libellarium/article/view/266/383 which I think should have been included in his paper.

**Comment by peterdesmet** [comment]
Would indeed be a good approach to separate the two (metadata review vs data review). For the latter, we'll try to automate that process at INBO with https://github.com/inbo/whip, especially for republications.

**Comment by sformel** [comment]
No, I don't see it in the GBIF-US IPT. Here is the code underlying the Associated Party dropdown menu:

```
<select name="eml.associatedParties[0].role" id="eml.associatedParties[0].role" size="1" class="form-select select2-hidden-accessible" data-select2-id="eml.associatedParties[0].role" tabindex="-1" aria-hidden="true">
                    <option value="" data-select2-id="18">
Select a role                    </option>
                    <option value="author" data-select2-id="19">
Author                    </option>
                    <option value="contentProvider" data-select2-id="20">
Content Provider                    </option>
                    <option value="custodianSteward" data-select2-id="21">
Custodian Steward                    </option>
                    <option value="distributor" data-select2-id="22">
Distributor                    </option>
                    <option value="editor" data-select2-id="23">
Editor                    </option>
                    <option value="metadataProvider" data-select2-id="24">
Metadata Provider                    </option>
                    <option value="originator" data-select2-id="25">
Originator                    </option>
                    <option value="owner" data-select2-id="26">
Owner                    </option>
                    <option value="pointOfContact" data-select2-id="27">
Point Of Contact                    </option>
                    <option value="principalInvestigator" data-select2-id="28">
Principal Investigator                    </option>
                    <option value="processor" data-select2-id="29">
Processor                    </option>
                    <option value="publisher" data-select2-id="30">
Publisher                    </option>
                    <option value="user" selected="selected" data-select2-id="10">
User                    </option>
                    <option value="programmer" data-select2-id="31">
Programmer                    </option>
                    <option value="curator" data-select2-id="32">
Curator                    </option>
            </select>
```

This is missing the last value (reviewer)  here: https://rs.gbif.org/vocabulary/gbif/agent_role.xml
